src/libs/ajax/leonardo/Runtimes.ts

```diff
@@ -28,7 +28,7 @@ import {
   RawListRuntimeItem,
 } from 'src/libs/ajax/leonardo/models/runtime-models';
 import { getConfig } from 'src/libs/config';
-import { CloudPlatform } from 'src/pages/billing/models/BillingProject';
+import { CloudProvider } from 'src/workspaces/utils';
```

Comment:
The CloudPlatform/CloudProvider situation is confusing, and I hope we come up with a unification.

But for now, I think it makes more sense for Leo to depend on the cloud provider exposed by workspaces instead of billing project. The billing project `CloudPlatform` allows for `UNKNOWN`, whereas Leo exists within a workspace that does have a known `CloudProvider`.

Reply:
Not necessarily for this PR, but generally... "Cloud platform" is the term used in the UI, so I would prefer to standardize on that name vs "cloud provider".

Reply:
A rename will touch a lot of files. I think it would make sense to do that at the time of the unification/simplification.